server/msgapi.py

```
@@ -109,6 +111,109 @@
+                    "attributes": {
+                        "channel": part
+                    }
+                })
+            else:
+                result.append({
+                    "insert": part
+                })
+                # 如果有at属性，则将其添加到insert的attributes中
+                try:
+                    if "attributes" in item:
+                        if "at" in item["attributes"]:
```

Ruff (0.11.9)

Use a single `if` statement instead of nested `if` statements

(SIM102)